### ISSUE TITLE
hardlight projector.

### ISSUE THREAD

**Issue by Mrcreepssss** [yogstation13/yogstation#1994]
so i recently had the idea of a hardlight projector, which for one could function like AI hologram only it can actually hold stuff, i'd imagine it would have very little storage and cant transfer stuff inbetween, (or maybe for an upgraded one.) and ofcourse it would have a cooldown, especially if destroyed. (idk if it should be global or local cooldown, depends on how it would be implemented.) i'd imagine it as either a MALF AI module or an RnD upgrade to standard holoprojectors.

i know this could be horrendously OP and i dont know if it would work well, just throwing out ideas.
you could also perhaps find other uses for hardlight projectors but this is what i thought of.

**Comment by Trojanplunder** [comment]
Doesn't goon have something similar, where the ai can control some little robot that has hands? I don't recall it being overpowered there. I could be wrong on this though, haven't played goon in a good while.

**Comment by Mrcreepssss** [comment]
i never play goon so i have no idea. and a little drone could work.

**Comment by Trojanplunder** [comment]
Yeah, if I recall right, the ai had two drones that it could jump into. They had really low health and moved rather slow, but allowed the ai to get certain things done that would normally require a crewmember or borg.

**Comment by AdamElTablawy** [comment]
Goon has that because their AI is exceptionally weak, since it cannot flood plasma. Also their 'drones' had 20 health and only have a screwdriver, wirecutter, multitool and wrench.

Also, no. The AI won't be getting hardlight projectors in any form. No AI holograms interacting with people or having hands.

**Comment by ShadowDeath6** [comment]
I like the idea of the AI interface module and stuff though
